operator/cmd/mesh/upgrade.go

```diff
@@ -139,8 +139,13 @@ func upgrade(rootArgs *rootArgs, args *upgradeArgs, l clog.Logger) (err error) {
 	if err != nil {
 		return err
 	}
+
 	// Generate IOPS parseObjectSetFromManifest
-	targetIOPSYaml, targetIOPS, err := GenerateConfig(args.inFilenames, ysf, args.force, nil, l)
+	config, err := defaultRestConfig(args.kubeConfigPath, args.context)
```

Comment:
since this is being changed, would you mind changing to K8sConfig to conform to the other commands? i think defaultRestConfig can be removed.

Reply:
Let's use a PR just for that, and not put the change into this one.

Reply:
Fixed here.